src/ontology/ro-edit.owl

```diff
@@ -6806,6 +6807,28 @@ SubClassOf(obo:BFO_0000015 ObjectAllValuesFrom(obo:RO_0002214 obo:BFO_0000015))
 
 SubClassOf(obo:CHEBI_50906 obo:PATO_0000001)
 
+# Class: obo:ExO_0000002 (exposure event)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CTD:curators") obo:IAO_0000115 obo:ExO_0000002 "An interaction between an exposure stressor and an exposure_receptor.")
+AnnotationAssertion(obo:IAO_0000116 obo:ExO_0000002 "Bill Duncan 2022-05-06:
+In the future, this class may become obsoleted if 'exposure event' is accepted as a class in COB. For now, we are using the class from ECTO.
+
+See COB issue https://github.com/OBOFoundry/COB/issues/192.")
+AnnotationAssertion(obo:IAO_0000116 obo:ExO_0000002 "Bill Duncan 2022-05-06:
+The `hasOBOnamespace` annotation was not taken from ECTO.
+
+I.e.:
+`hasOBOnamespace` exposure event")
+AnnotationAssertion(obo:IAO_0000116 obo:ExO_0000002 "Bill Duncan 2022-05-06:
+This term was placed in RO after in order to avoid confusion between ECTO:'exposure process' and RO:'exposure event or process'. Due to ECTO importing RO, both classes were present in ECTO. This makes editor unsure which class to use in ECTO. 
+
+See RO issue https://github.com/oborel/obo-relations/issues/599.")
+AnnotationAssertion(oboInOwl:created_by obo:ExO_0000002 "cmattin")
```

Comment:
This is the reason the QC is failing now: not using ORCID. I understand this is still a WIP.

Reply:
@matentzn  @anitacaron The `oboInOwl:created_by` is an old annotation. Can we get this pushed though? I don't. have the resources to fix all past non-conforming annotations, and this PR is quite old.

Reply:
Yes. I understand that.  
```AnnotationAssertion(oboInOwl:created_by obo:ExO_0000002 "cmattin")```
I don't know `cmattin`. If anyone does, can they provide the ORCID, delete the annotation, or push this PR through. RO development doesn't need to be held up by ORCID annotations for creators we don't know.

Reply:
This AnnotationAssertion is new to this PR, so you can remove it.